README.md

```diff
@@ -1,20 +1,61 @@
 # SQL Datastore
 
+[![CircleCI](https://circleci.com/gh/ipfs/go-ds-sql.svg?style=shield)](https://circleci.com/gh/ipfs/go-ds-sql)
+[![Coverage](https://codecov.io/gh/ipfs/go-ds-sql/branch/master/graph/badge.svg)](https://codecov.io/gh/ipfs/go-ds-sql)
+[![Standard README](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg)](https://github.com/RichardLitt/standard-readme)
+[![GoDoc](http://img.shields.io/badge/godoc-reference-5272B4.svg)](https://godoc.org/github.com/ipfs/go-ds-sql)
+[![golang version](https://img.shields.io/badge/golang-%3E%3D1.14.0-orange.svg)](https://golang.org/)
+[![Go Report Card](https://goreportcard.com/badge/github.com/ipfs/go-ds-sql)](https://goreportcard.com/report/github.com/ipfs/go-ds-sql)
+
 An implementation of [the datastore interface](https://github.com/ipfs/go-datastore)
 that can be backed by any sql database.
 
+## Install
+
+```sh
+go get github.com/ipfs/go-ds-sql
+```
+
 ## Usage
 
+Ensure a database is created and a table exists with `key` and `data` columns. For example, in PostgreSQL you can create a table with the following structure (replacing `table_name` with the name of the table the datastore will use - by default this is `blocks`):
+
+```sql
+CREATE TABLE IF NOT EXISTS table_name (key TEXT NOT NULL UNIQUE, data BYTEA)
```

Comment:
I wonder if this is equivalent to having keys be the hash of the actual key in terms of how the database actually manages it. I know that text columns are typically held in lookaside tables because they can be too wide to fit in a normal column and so end up with additional pointer follows / are generally suboptimal for use as keys directly. In this case though i don't know if there's a manual setup we can do that really saves much.